server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/pipeline/ProjectStepResultChecker.java

```diff
@@ -139,7 +140,16 @@ private void checkPreCheckStepResult() {
         }
     }
 
-    private boolean checkProjectFinished() {
+    private boolean isProjectReleased() {
```

Comment:
delete, releasing status mapping to released, seems hard to understand ?

Reply:
ok, these status can mapping to `Destroyed`